provision.go

```diff
@@ -20,10 +21,14 @@ func provision(ctx context.Context, deps k6deps.Dependencies, opts *Options) (st
 		return "", err
 	}
 
+	slog.Debug("downloading binary", "build service URL: ", opts.BuildServiceURL)
```

Comment:
How about increasing the severity of the log, to INFO? Probably it's something that takes some time, so we notify all users that something is happaning behind the scene even wwithout `-v`. WDYT?

```suggestion
	slog.Info("downloading k6 binary...", "build service URL: ", opts.BuildServiceURL)
```

Reply:
I had it this way, but I think for the regular user it is more valuable to know we are downloading the binary than where we are downloading it. This is way I separated the information into two log entries, one info and the other debug.

Reply:
Yes, I do agree that the source (builder URL) could be omitted, but I'm not sure if I got what another (info) message than

Reply:
Not following your last message. I meant this `INFO` log entry is generated 
```
INFO[0000] provisioning k6 binary                        app=k6exec
```

I agree this is not the most user-friendly message, but we can add here more datils if you think that will help the user in normal circumstances.

Reply:
I meant I wasn't sure that `provisioning k6 binary` (or now `fetching k6 binary`) is the one that covering this need. I think after adjustment it all looks good :+1: